<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Peresvet</class>
 <widget class="QWidget" name="Peresvet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Пересвет Панель</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>701</width>
     <height>501</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Главная</string>
    </attribute>
    <widget class="QPushButton" name="run_me">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>410</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgba(0, 255, 0, 120);
border: 2px solid black;
</string>
     </property>
     <property name="text">
      <string>Запуск</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>10</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"> background-color: rgba(255, 255, 255, 230); color: black;
 padding: 10px;
border-radius: 10px;</string>
     </property>
     <property name="text">
      <string>Автозапуск</string>
     </property>
    </widget>
    <widget class="QPushButton" name="restart_me">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>410</y>
       <width>93</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgba(255, 255, 0, 120);
border: 2px solid black;
</string>
     </property>
     <property name="text">
      <string>Перезапуск</string>
     </property>
    </widget>
    <widget class="QPushButton" name="stop_me">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>410</y>
       <width>93</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgba(255, 0, 0, 120);
border: 2px solid black;
</string>
     </property>
     <property name="text">
      <string>Остановить</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
        background-color: rgba(255, 255, 255, 230);
        color: black;
		border: 2px solid black;
        padding: 10px;
        border-radius: 10px;
    }
QPushButton:hover {
    background-color: rgba(230, 230, 230, 230);
}
QPushButton:pressed {
    background-color: rgba(200, 200, 200, 230);
}
QLabel {
background-color: rgba(255, 255, 255, 230);
color: black;
padding-left: 5px;
}</string>
     </property>
     <property name="text">
      <string>Открыть</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>240</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Molodo-font</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 10px;</string>
     </property>
     <property name="text">
      <string>ADMINER</string>
     </property>
    </widget>
    <widget class="QPushButton" name="open_adminer">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>290</y>
       <width>93</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
        background-color: rgba(255, 255, 255, 230);
        color: black;
        padding: 10px;
		border: 2px solid black;
        border-radius: 10px;
    }
QPushButton:hover {
    background-color: rgba(230, 230, 230, 230);
}
QPushButton:pressed {
    background-color: rgba(200, 200, 200, 230);
}
QPushButton:enabled {
    background-color: rgba(255, 255, 255, 230);
}
QPushButton:disabled {
    background-color: rgba(130, 130, 130, 230);
}</string>
     </property>
     <property name="text">
      <string>Открыть</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Molodo-font</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 10px;</string>
     </property>
     <property name="text">
      <string>phpMyAdmin</string>
     </property>
    </widget>
    <widget class="QPushButton" name="open_phpmyadmin">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>93</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
        background-color: rgba(255, 255, 255, 230);
        color: black;
        padding: 10px;
		border: 2px solid black;
        border-radius: 10px;
    }
QPushButton:hover {
    background-color: rgba(230, 230, 230, 230);
}
QPushButton:pressed {
    background-color: rgba(200, 200, 200, 230);
}
QPushButton:enabled {
    background-color: rgba(255, 255, 255, 230);
}
QPushButton:disabled {
    background-color: rgba(130, 130, 130, 230);
}</string>
     </property>
     <property name="text">
      <string>Открыть</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Molodo-font</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 10px;</string>
     </property>
     <property name="text">
      <string>Проекты</string>
     </property>
    </widget>
    <widget class="QLabel" name="icon1">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>370</y>
       <width>91</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Настройки модулей</string>
    </attribute>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 3px;</string>
     </property>
     <property name="text">
      <string>Веб-серверы</string>
     </property>
    </widget>
    <widget class="QComboBox" name="apache_list">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>201</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="nginx_list">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>201</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="php_list">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>170</y>
       <width>201</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="postgresql_list">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>270</y>
       <width>201</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="mysql_list">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>320</y>
       <width>201</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="redis_list">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>370</y>
       <width>201</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="server_type">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>20</y>
       <width>201</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 3px;</string>
     </property>
     <property name="text">
      <string>Выключено</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="apache_checkbox">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>70</y>
       <width>121</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QCheckBox::indicator {
    width:  25px;
    height: 25px;
}
QCheckBox::indicator:checked {
    image: url(images/checkbox_active.png);
}
QCheckBox::indicator:unchecked  {
    image: url(images/checkbox_deactive.png);
}
QCheckBox::indicator:disabled  {
    image: url(images/checkbox_disabled.png);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QCheckBox" name="nginx_checkbox">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>120</y>
       <width>121</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QCheckBox::indicator {
    width:  25px;
    height: 25px;
}
QCheckBox::indicator:checked {
    image: url(images/checkbox_active.png);
}
QCheckBox::indicator:unchecked  {
    image: url(images/checkbox_deactive.png);
}
QCheckBox::indicator:disabled  {
    image: url(images/checkbox_disabled.png);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QCheckBox" name="postgresql_checkbox">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>270</y>
       <width>121</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QCheckBox::indicator {
    width:  25px;
    height: 25px;
}
QCheckBox::indicator:checked {
    image: url(images/checkbox_active.png);
}
QCheckBox::indicator:unchecked  {
    image: url(images/checkbox_deactive.png);
}
QCheckBox::indicator:disabled  {
    image: url(images/checkbox_disabled.png);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QCheckBox" name="mysql_checkbox">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>320</y>
       <width>121</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QCheckBox::indicator {
    width:  25px;
    height: 25px;
}
QCheckBox::indicator:checked {
    image: url(images/checkbox_active.png);
}
QCheckBox::indicator:unchecked  {
    image: url(images/checkbox_deactive.png);
}
QCheckBox::indicator:disabled  {
    image: url(images/checkbox_disabled.png);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QCheckBox" name="redis_checkbox">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>370</y>
       <width>121</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QCheckBox::indicator {
    width:  25px;
    height: 25px;
}
QCheckBox::indicator:checked {
    image: url(images/checkbox_active.png);
}
QCheckBox::indicator:unchecked  {
    image: url(images/checkbox_deactive.png);
}
QCheckBox::indicator:disabled  {
    image: url(images/checkbox_disabled.png);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>150</y>
       <width>201</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 3px;</string>
     </property>
     <property name="text">
      <string>PHP</string>
     </property>
    </widget>
    <widget class="QLabel" name="icon2">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>370</y>
       <width>91</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="server_type_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>201</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 3px;</string>
     </property>
     <property name="text">
      <string>apache</string>
     </property>
    </widget>
    <widget class="QLabel" name="server_type_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>201</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 3px;</string>
     </property>
     <property name="text">
      <string>nginx</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 3px;</string>
     </property>
     <property name="text">
      <string>Базы данных</string>
     </property>
    </widget>
    <widget class="QLabel" name="server_type_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>250</y>
       <width>201</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 3px;</string>
     </property>
     <property name="text">
      <string>postgresql</string>
     </property>
    </widget>
    <widget class="QLabel" name="server_type_5">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>300</y>
       <width>201</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 3px;</string>
     </property>
     <property name="text">
      <string>mysql</string>
     </property>
    </widget>
    <widget class="QLabel" name="server_type_6">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>350</y>
       <width>201</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 3px;</string>
     </property>
     <property name="text">
      <string>redis</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
